tests/unit_tests/test_arctan2.py

```diff
@@ -0,0 +1,74 @@
+import json
+import math
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def arctan2_process_code():
+    return load_process_code("arctan2")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        # test generated from examples at
+        # https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2
+        # -0 has to be passed as a float so that the python handles it as a -0, not 0
```

Comment:
Ok, but this would make sense only if you would write your own implementation of atan2. Now you are just testing some built in function. I think it is safe to assume that Math functions working correctly and  there is no need to write so many tests for them.